### ISSUE TITLE
"to5 processor compilation failed" when JSX contains any "/" in render method

### ISSUE THREAD

**Issue by hugoeanogueira** [6to5/6to5#398]
6to5 fails compiling because of the slash ("/") after a tag.
This doesn't happen if we leave a space or a newline between them.

```
/** @jsx React.DOM */

var Component = React.createClass({
  render: function () {
    return (
      <h1>/Authenticate</h1>
    );
  }
});

React.render(<Component />, document.body);
```

**Comment by sebmck** [comment]
Can you report this upstream at [acorn-jsx](https://github.com/RReverser/acorn-jsx) too please

**Comment by hugoeanogueira** [comment]
Issue [#12](https://github.com/RReverser/acorn-jsx/issues/12).
Done!

**Comment by RReverser** [comment]
Fixed in acorn-jsx.

**Comment by sebmck** [comment]
Thanks @RReverser. Released as of 2.10.0
